tyssue/core/history.py

```diff
@@ -299,18 +299,23 @@ def time_stamps(self):
             times = file.select("vert", columns=["time"])["time"].unique()
         return times
 
-    def record(self, to_record=None, time_stamp=None):
+    def record(self, to_record=None, time_stamp=None, sheet=None):
         """Appends a copy of the sheet datasets to the history HDF file.
 
         Parameters
         ----------
         to_report : Deprecated - list of strings
             the datasets from self.sheet to be saved
+        sheet: a :class:`Sheet` object which we want to record. This argument is used
+        to convert several HF5 files of one simulation to only one HF5 files.
```

Comment:
I'd rather write (maybe more generally):
this argument can be used if the sheet object is different at each time point

Reply:
I'll change.